CMSgov/mint-app
# [EASI-2423] Operational Solution Plan Document Links

# EASI-2423

## Changes and Description

Added endpoints to Create and Remove operational solution plan document links

- CreatePlanDocumentSolutionLinks
  - Creates an array of plan document solution links
  - Returns the resulting array of plan document solution links with the generated ID

- RemovePlanDocumentSolutionLink
  - Removes a plan document solution link by UUID
  
- PlanDocumentSolutionLinks
  - Gets a list of document solution links associated with a model plan ID



## How to test this change

Run the following endpoints and observe that they function as expected:

- createPlanDocumentSolutionLinks
- removePlanDocumentSolutionLink

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
